controllers/topic-controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectAllTopics } = require("../models/topic-models");
+
+exports.getTopics = (req, res, next) => {
+  selectAllTopics().then((topics) => {
+    res.status(200).send({ topics });
+  });
```

Comment:
I'd like to see a catch here on the controller, for if anything was to go wrong with the model.
Don't necessarily need to handle the error yet but catch's on a controller should really be default.